src/components/Upload/UploadManager.tsx

```diff
@@ -0,0 +1,235 @@
+import {
+  Dialog,
+  DialogContent,
+  IconButton,
+  makeStyles,
+  DialogTitle as MuiDialogTitle,
+  Paper,
+  PaperProps,
+  Typography,
+} from '@material-ui/core';
+import {
+  Close as CloseIcon,
+  Maximize as MaximizeIcon,
+  Minimize as MinimizeIcon,
+} from '@material-ui/icons';
+import clsx from 'clsx';
+import React, {
+  FC,
+  memo,
+  useCallback,
+  useLayoutEffect,
+  useRef,
+  useState,
+} from 'react';
+import Draggable, { DraggableData, DraggableEvent } from 'react-draggable';
+import { useWindowSize } from 'react-use';
+import { useSession } from '../Session';
+import { useUploadManager } from './UploadContext';
+
+const PAPER_WIDTH = 480;
+
+const useStyles = makeStyles(({ palette, spacing }) => ({
+  root: {
+    pointerEvents: 'none',
+  },
+  paper: {
+    margin: 0,
+    pointerEvents: 'auto',
+    position: 'absolute',
+    top: 0,
+    left: 0,
+    width: `${PAPER_WIDTH}px`,
```

Comment:
FYI pixels are assumed for numbers.
```suggestion
    width: PAPER_WIDTH,
```